src/nz-tour.js

```diff
@@ -497,8 +497,12 @@
                         showNext: steps[step].showNext === undefined ? config.showNext : steps[step].showNext,
                         showPrevious: steps[step].showPrevious === undefined ? config.showPrevious : steps[step].showPrevious
                     };
-                    //Don't mess around with angular sanitize for now. Add compile and sanitize later...
-                    els.innerContent.html(steps[step].content);
+                    var stepHtml =  steps[step].content;
+                    // Compile the step definition html so ng-click works as expected.
+                    var compiledHtml = $compile(stepHtml)($scope);
+                    els.innerContent.html('');
+                    els.innerContent.append(compiledHtml);
```

Comment:
Thanks @bmartinelle. The dist/src sync was my fault :grin:

Just a quick question, why not just `els.innerContent.html(compiledHtml)` instead of clearing + appending?